mautic/mautic
# #TPROD-329 - migrate UserBundle to twig templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. In the top right corner of the screen, open the "Settings" menu and click "Users".
3. Test every single functionality on this page. Creating/editing/cloning webhooks, filtering, pagination, etc. - all functionality should work exactly like it did before.
4.  In the top right corner of the screen, open the "Settings" menu and click "Roles".
5. Test every single functionality on this page. Creating/editing/cloning webhooks, filtering, pagination, etc. - all functionality should work exactly like it did before.
